lib/ansible/modules/cloud/vmware/vmware_vm_shell.py

```diff
@@ -16,10 +16,10 @@
 DOCUMENTATION = '''
 ---
 module: vmware_vm_shell
-short_description: Execute a process in VM
+short_description: Starts a program in the guest operating system
 description:
-    - Start a program in a VM without the need for network connection
-version_added: 2.1
+    - Module allows user to run common system administration commands in the guest operating system.
+version_added: "2.1"
 author: "Ritesh Khadgaray (@ritzk)"
```

Comment:
Make author a list, and drop the quotes.

Reply:
Done